Cargo.toml

```diff
@@ -1,60 +1,11 @@
-[package]
-name = "tox"
-version = "0.0.10"
-authors = [
-  "Zetok Zalbavar <zetok@openmailbox.org>",
-  "Roman Proskuryakov <humbug@deeptown.org>",
-  "Namsoo CHO <nscho66@gmail.com>",
-  "Evgeny Kurnevsky <kurnevsky@gmail.com>",
+[workspace]
+members = [
+    "tox",
+    "binary_io",
```

Comment:
Should it be prefixed with `tox_`?

Reply:
It's not really tox specific, though I can prefix it with `tox_` if necessary.

Reply:
It'll have this name on cartes.io which might be confusing, especially if it contains tox-specific encoding for `IpAddr`.